functions.php

```diff
@@ -39,15 +41,16 @@ function getPokemons(PDO $db): array {
 function displayPokemon(array $pokemons): string {
     $result = '';
     foreach ($pokemons as $pokemon) {
-        if (array_key_exists('img_source', $pokemon) && array_key_exists('pokedex_no', $pokemon) && array_key_exists('name', $pokemon) && array_key_exists('type', $pokemon)) {
+        if (array_key_exists('img_source', $pokemon) &&
```

Comment:
still not correct, check my last comment. first and last condition should be on their own line

Reply:
Sorry didn't spot that, all sorted.